UnitsNet/Scripts/UnitDefinitions/MassFlow.json

```diff
@@ -0,0 +1,31 @@
+{
+    "Name": "MassFlow",
+    "BaseUnit": "KilogramPerSecond",
+    "XmlDoc": "",
+    "Units": [
+        {
+            "SingularName": "KilogramPerSecond",
+            "PluralName": "KilogramsPerSecond",
+            "FromUnitToBaseFunc": "x",
+            "FromBaseToUnitFunc": "x",
+            "Localization": [
+                {
+                    "Culture": "en-US",
+                    "Abbreviations": ["kg/s"]
+                }
+            ]
+        },
+		{
+            "SingularName": "TonnePerDay",
+            "PluralName": "TonnesPerDay",
+            "FromUnitToBaseFunc": "x*86.4000",
+            "FromBaseToUnitFunc": "x/86.4000",
```

Comment:
I know, the only "failsafe" is that the unit tests asks you to implement each case and the wording of the tests were supposed to be more or less googlable so you'd get it right, but I've even accidentally swapped units myself. Easy to do. Not sure how we could make it more fool proof. I've double-checked all your conversions, but it doesn't hurt if you do too.